apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/trace/SpanLayer.java

```diff
@@ -19,7 +19,7 @@
 package org.apache.skywalking.apm.agent.core.context.trace;
 
 public enum SpanLayer {
-    DB(1), RPC_FRAMEWORK(2), HTTP(3), MQ(4), CACHE(5);
+    DB(1), RPC_FRAMEWORK(2), HTTP(3), MQ(4), CACHE(5), CUSTOM_ENHANCE(9);
```

Comment:
This is illegal.

Reply:
Does this mean that new enumeration values are not allowed to be added,  or must the code of the added enumeration value be incremented（CUSTOM_ENHANCE(6)  ？）